private/protocol/xml/xmlutil/unmarshal_bench_test.go

```diff
@@ -0,0 +1,171 @@
+package xmlutil_test
+
+import (
+	"bytes"
+	"encoding/xml"
+	"io/ioutil"
+	"net/http"
+	"testing"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/private/protocol/xml/xmlutil"
+	"github.com/aws/aws-sdk-go-v2/service/ec2"
+)
+
+type DataOutput struct {
+	_ struct{} `type:"structure"`
+
+	FooEnum string `type:"string" enum:"true"`
+
+	ListEnums []string `type:"list"`
+}
+
+func BenchmarkXMLUnmarshal_Simple(b *testing.B) {
+	b.ResetTimer()
+	for i := 0; i < b.N; i++ {
+		req := getXMLResponseSimple()
+		xmlutil.UnmarshalXML(req.Data, xml.NewDecoder(req.HTTPResponse.Body), "")
+	}
+}
+
+func BenchmarkXMLUnmarshal_Complex(b *testing.B) {
+	b.ResetTimer()
+	for i := 0; i < b.N; i++ {
+		req := getXMLResponseComplex()
+		xmlutil.UnmarshalXML(req.Data, xml.NewDecoder(req.HTTPResponse.Body), "")
+	}
+}
+
+func getXMLResponseSimple() *aws.Request {
+	buf := bytes.NewReader([]byte("<OperationNameResponse><FooEnum>foo</FooEnum><ListEnums><member>0</member><member>1</member></ListEnums></OperationNameResponse>"))
+	req := aws.Request{Data: &DataOutput{}, HTTPResponse: &http.Response{Body: ioutil.NopCloser(buf)}}
+	return &req
+}
+
+func getXMLResponseComplex() *aws.Request {
+	buf := bytes.NewReader([]byte(`2019/07/01 15:01:44 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
The leading `2019/07/01 15:01:44 ` should be removed from this test string,

Reply:
The leading string "2019/07/01 15:01:44" has been removed from private/protocol/xml/xmlutil/unmarshal_bench_test.go. Please check if there are further issues.